### PR TITLE
Add Livecoding

### PR DESCRIPTION
**Pull request by lvm**
**https://github.com/lvm/awesome-livecoding/**

**This lists brings together everything related to the LiveCoding practice, mainly in the form of arts (music, visuals) but all contributions related to the topic are welcome.**

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:
	- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
	- It's the result of hard work and the best I could possibly produce.
	- Non-generated Markdown file in a GitHub repo.
	- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
	- Not a duplicate.
	- Only has awesome items. Awesome lists are curations of the best, not everything.
	- Includes a project logo/illustration whenever possible.
		- Placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
		- The image should link to the project website or any relevant website.
		- The image should be high-DPI. Set it to maximum half the width of the original image.
	- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
	- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
		- Should be placed on the right side of the readme heading.
		- Should link back to this list.
	- Has a Table of Contents section.
		- Should be named `Contents`, not `Table of Contents`.
		- Should be the first section in the list.
		- Should only have one level of sub-lists, preferably none.
	- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
		- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
		- If you use a license badge, it should be SVG, not PNG.
	- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
		- The file should be named `contributing.md`. Casing is up to you.
	- Has consistent formatting and proper spelling/grammar.
		- Each link description starts with an uppercase character and ends with a period.<br>Example: `- [AVA](…) - JavaScript test runner.`
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
	- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Go to the top and read it again.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Please read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) more closely. Multiple points not followed from it. Try going through the list point for point and check that you follow it. :)

**Comment by lvm** [comment]
Hi there,
the only points i could think of that are not being followed are:

* Should be named Contents, not Table of Contents. **(fixed)**
* The file should be named contributing.md. Casing is up to you. **(fixed)**
* (awesome logo) Should be placed on the right side of the readme heading. **(fixed)**

On the other hand, there are these:

* Only has awesome items. Awesome lists are curations of the best, not everything.

the list is already compilaton of the best (which might be most of the projects available but not all)

* Includes a project logo/illustration whenever possible. 

don't have, nor care about it.

* Each link description starts with an uppercase character and ends with a period.

I could fix the capitalization of the descriptions but disagree about ending with a period.

Let me know what else i'm missing.

Thanks.

File: readme.md [hunk 1]
```diff
@@ -535,6 +535,7 @@
 - [Web Design](https://github.com/nicolesaidy/awesome-web-design) - For digital designers.
 - [JMeter](https://github.com/aliesbelik/awesome-jmeter) - Load testing and performance measurement tool.
 - [Creative Coding](https://github.com/terkelg/awesome-creative-coding) - Programming something expressive instead of something functional.
+- [Livecoding](https://github.com/lvm/awesome-livecoding) - All things livecoding.
```
Comment (sindresorhus) [inline readme.md]: > The entry in the Awesome list should:
Include a short description about the project/theme of the list. **It should not describe the list itself.**

:arrow_up:

**Comment by sindresorhus** [comment]
- `livecoding` is not an actual word, it's `live coding`.
- The `Windows | OS X | GNU/Linux C++ FLOSS` tag things make the list quite. I would try some different formatting. Maybe try putting them on a separate line.
- `OS X` => `macOS`

---

> but disagree about ending with a period.

It's the same for everyone.

**Comment by lvm** [comment]
> The Windows | OS X | GNU/Linux C++ FLOSS tag things make the list quite. I would try some different formatting. Maybe try putting them on a separate line.

quite? i'm not sure i follow what you mean.

The reason why the descriptions doesn't begin capitalized / end with a period is because https://en.oxforddictionaries.com/punctuation/bullet-points not just a whim.

> Include a short description about the project/theme of the list. It should not describe the list itself.

A short description of the theme is "all things live coding" ([partly as a joke](https://i.imgur.com/JWdLbfi.png)), it isn't describing the list but the topic :-) 
If i were describing the list i would've written "languages, libraries, videos and stuff to read about live coding". 
Anyway, what should i write? "For people who (live) code"? At least the "Web Design" list seem to be ok and has a similar description.

**Comment by sindresorhus** [comment]
> quite? i'm not sure i follow what you mean.

It was missing the last word:

The `Windows | OS X | GNU/Linux C++ FLOSS` tag thing makes the list quite messy. I would try some different formatting. Maybe try putting them on a separate line.

---

> The reason why the descriptions doesn't begin capitalized / end with a period is because en.oxforddictionaries.com/punctuation/bullet-points not just a whim.

From your linked site:

> If the text that follows the bullet point is not a proper sentence, it doesn’t need to begin with a capital letter, nor end with a period.

So your argument is only true when the bullet points are not proper sentences, but they should be proper sentences, so your argument doesn't apply. Anyway, I'm not interested in discussing this. As I said, it's the same for everyone.

---

> A short description of the theme is "all things live coding" (partly as a joke), it isn't describing the list but the topic :-)
If i were describing the list i would've written "languages, libraries, videos and stuff to read about live coding".
Anyway, what should i write? "For people who (live) code"? At least the "Web Design" list seem to be ok and has a similar description.

Just look at the other descriptions. It should describe **what** live coding is.

**Comment by lvm** [comment]
> The `Windows | OS X | GNU/Linux C++ FLOSS` tag thing makes the list quite messy. I would try some different formatting. Maybe try putting them on a separate line.
Something like this ?

* afterglow, a live-coding lighting controller, building on the Open Lighting Architecture with Clojure and bits of Overtone 
`Windows | macOS | GNU/Linux` `lisp` `FLOSS` 
* Alda, a music programming language for musicians. 
`Windows | macOS | GNU/Linux` `FLOSS`
* ChucK, Strongly-timed, Concurrent, and On-the-fly Music Programming Language. 
`Windows | macOS | GNU/Linux` `FLOSS`
* Conductive, a set of Haskell libraries for livecoding and real-time music applications. 
`Windows | macOS | GNU/Linux` `haskell` `Supercollider` `FLOSS`

a/ On one hand, it does looks a bit nicer
b/ You loose the hability to filter languages (`grep lisp awesome-livecoding.txt`)

I still prefer the option b.

>   Just look at the other descriptions. It should describe what live coding is.

does this works? "Live coding (sometimes referred to as 'on-the-fly programming', 'just in time programming' and 'conversational programming') is a performing arts form and a creativity technique centred upon the writing of source code and the use of interactive programming in an improvised way" 
Or perhaps a shorter version?

Have a nice day.

**Comment by sindresorhus** [comment]
Hey, sorry for the lack of response, please see #1363.

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/awesome/pull/1049/files#r143970439 is not fixed. Please re-read my comment.

**Comment by sindresorhus** [comment]
> b/ You loose the hability to filter languages (`grep lisp awesome-livecoding.txt`)

That's irrelevant. Lists should be optimized for reading by humans, not for `grep`.

**Comment by sindresorhus** [comment]
> "Live coding (sometimes referred to as 'on-the-fly programming', 'just in time programming' and 'conversational programming') is a performing arts form and a creativity technique centred upon the writing of source code and the use of interactive programming in an improvised way"
Or perhaps a shorter version?

That's a lot of words to say very little. I think you could come up with something better than the Wikipedia intro.

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/toplap/awesome-livecoding
✖ Linting

  README.md:26:1
  ✖    1:1    Missing or invalid Table of Contents                                             remark-lint:awesome/toc
  ✖   11:3    Invalid list item link URL                                                       remark-lint:awesome/list-item
  ✖   12:3    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   12:5    Invalid list item link URL                                                       remark-lint:awesome/list-item
  ✖   13:3    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   13:5    Invalid list item link URL                                                       remark-lint:awesome/list-item
  ✖   14:3    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   14:5    Invalid list item link URL                                                       remark-lint:awesome/list-item
  ✖   15:3    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   15:5    Invalid list item link URL                                                       remark-lint:awesome/list-item
  ✖   16:3    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   16:5    Invalid list item link URL                                                       remark-lint:awesome/list-item
  ✖   17:3    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   17:5    Invalid list item link URL                                                       remark-lint:awesome/list-item
  ✖   18:3    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   18:5    Invalid list item link URL                                                       remark-lint:awesome/list-item
  ✖   19:3    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   19:5    Invalid list item link URL                                                       remark-lint:awesome/list-item
  ✖   20:3    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   20:5    Invalid list item link URL                                                       remark-lint:awesome/list-item
  ✖   21:3    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   21:5    Invalid list item link URL                                                       remark-lint:awesome/list-item
  ✖   22:3    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   22:5    Invalid list item link URL                                                       remark-lint:awesome/list-item
  ✖   23:3    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   23:5    Invalid list item link URL                                                       remark-lint:awesome/list-item
  ✖   26:1    Don’t add a trailing ? to headings                                               remark-lint:no-heading-punctuation
  ✖   35:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   35:32   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   42:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   42:54   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   43:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   43:44   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   44:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   44:42   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   45:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   45:38   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   46:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   46:65   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   47:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   47:33   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   48:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   48:45   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   49:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   49:42   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   50:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   50:31   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   51:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   51:47   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   52:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   52:40   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   53:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   53:37   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   54:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   54:40   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   55:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   55:45   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   56:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   56:52   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   57:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   57:56   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   58:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   58:49   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   59:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   59:41   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   60:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   60:43   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   61:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   61:38   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   62:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   62:52   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   63:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   63:56   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   64:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   64:53   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   65:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   65:48   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   66:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   66:35   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   67:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   67:44   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   68:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   68:52   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   69:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   69:51   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   70:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   70:50   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   71:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   71:42   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   72:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   72:60   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   73:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   73:35   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   74:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   74:25   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   75:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   75:28   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   76:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   76:52   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   77:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   77:39   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   84:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   84:46   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   85:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   85:44   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   86:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   86:53   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   87:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   87:51   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   88:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   88:58   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   89:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   89:59   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   90:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   90:57   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   91:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   91:48   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   92:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   92:54   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   93:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   93:37   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   94:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   94:56   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   95:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   95:63   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   96:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   96:39   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   97:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   97:67   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   97:67   Text "MacOS" should be written as "macOS"                                        remark-lint:awesome/spell-check
  ✖   98:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   98:51   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖   99:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖   99:42   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  100:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  100:48   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  101:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  101:46   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  102:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  102:48   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  103:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  103:49   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  104:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  104:51   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  105:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  105:42   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  106:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  106:49   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  107:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  107:49   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  108:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  108:44   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  109:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  109:49   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  110:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  110:47   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  111:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  111:53   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  112:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  112:63   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  113:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  113:59   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  114:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  114:44   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  115:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  115:56   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  116:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  116:63   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  117:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  117:64   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  118:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  118:60   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  119:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  119:54   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  120:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  120:50   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  121:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  121:59   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  122:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  122:54   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  123:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  123:52   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  124:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  124:44   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  125:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  125:54   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  126:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  126:42   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  127:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  127:27   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  128:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  128:54   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  129:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  129:51   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  134:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  134:76   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  135:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  135:88   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  136:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  136:76   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  137:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  137:72   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  138:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  138:109  List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  139:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  139:68   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  140:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  140:53   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  141:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  141:76   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  142:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  142:4    Text "Youtube" should be written as "YouTube"                                    remark-lint:awesome/spell-check
  ✖  142:116  List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  143:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  143:4    Text "Youtube" should be written as "YouTube"                                    remark-lint:awesome/spell-check
  ✖  143:90   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  144:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  144:102  List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  145:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  145:70   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  146:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  146:106  List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  151:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  151:62   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  152:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  152:91   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  153:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  153:114  List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  154:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  154:114  List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  155:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  155:74   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  156:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  156:63   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  157:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  157:118  List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  158:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  158:69   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  159:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  159:113  List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  160:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  160:78   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  161:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  161:170  List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  166:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  166:36   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  167:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  167:41   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  168:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  168:64   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  172:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  172:43   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  173:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  173:37   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  174:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  174:41   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  175:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  175:30   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  179:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  179:33   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  180:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  180:38   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  181:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  181:36   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  182:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  182:56   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  183:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  183:54   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  188:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  188:80   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  189:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  189:94   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  190:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  190:60   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  191:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  191:87   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  192:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  192:108  List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  193:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  193:68   List item link and description must be separated with a dash                     remark-lint:awesome/list-item
  ✖  224:1    Don’t add a trailing ? to headings                                               remark-lint:no-heading-punctuation
  ✖  226:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  227:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  228:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  229:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  230:1    Marker style should be -                                                         remark-lint:unordered-list-marker-style
  ✖  237:114  Link to http://contributor-covenant.org/version/1/4/code_of_conduct.txt is dead  remark-lint:no-dead-urls
```

https://github.com/sindresorhus/awesome-lint

**Comment by lvm** [comment]
is this still open? ffs.

**Pull request closed without merging**